.nf-core.yml => linter error

### Description of the bug

https://github.com/nf-core/scnanoseq/blob/master/.nf-core.yml includes:

```
  pipeline_todos:
    - README.md
    - main.nf
```

`pipeline_todos` should be a boolean:

```
AssertionError: Config file '.nf-core.yml' is invalid
lint: Input should be a valid boolean
input: ['README.md', 'main.nf']
```

### Command used and terminal output

```console

```

### Relevant files

_No response_

### System information

_No response_

--- a/.github/workflows/linting.yml
+++ b/.github/workflows/linting.yml
@@ -14,10 +14,10 @@ jobs:
   pre-commit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
 
       - name: Set up Python 3.12
-        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5
+        uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5
         with:
           python-version: "3.12"
 
@@ -31,12 +31,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out pipeline code
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
 
       - name: Install Nextflow
         uses: nf-core/setup-nextflow@v2
 
-      - uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5
+      - uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5
         with:
           python-version: "3.12"
           architecture: "x64"
@@ -74,7 +74,7 @@ jobs:
 
       - name: Upload linting log file artifact
         if: ${{ always() }}
-        uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4
+        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4
         with:
           name: linting-logs
           path: |

--- a/.github/workflows/linting.yml
+++ b/.github/workflows/linting.yml
@@ -6,7 +6,6 @@ on:
   push:
     branches:
       - dev
-      - template_update
   pull_request:
   release:
     types: [published]
@@ -15,10 +14,10 @@ jobs:
   pre-commit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
 
       - name: Set up Python 3.12
-        uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5
+        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5
         with:
           python-version: "3.12"
 
@@ -32,12 +31,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out pipeline code
-        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
 
       - name: Install Nextflow
         uses: nf-core/setup-nextflow@v2
 
-      - uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5
+      - uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5
         with:
           python-version: "3.12"
           architecture: "x64"
@@ -75,7 +74,7 @@ jobs:
 
       - name: Upload linting log file artifact
         if: ${{ always() }}
-        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4
+        uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4
         with:
           name: linting-logs
           path: |

--- a/.gitignore
+++ b/.gitignore
@@ -6,9 +6,9 @@ results/
 testing/
 testing*
 *.pyc
+null/
 test.sh
 params.yml
 samplesheet.csv
 *.swp
 input*
-null/

--- a/.nf-core.yml
+++ b/.nf-core.yml
@@ -4,7 +4,6 @@ lint:
     - lib/NfcoreTemplate.groovy
     - docs/images/nf-core-scnanoseq_logo_dark.png
     - docs/images/nf-core-scnanoseq_logo_light.png
-    - .gitignore
   template_strings: false
 nf_core_version: 3.0.2
 org_path: null

--- a/.nf-core.yml
+++ b/.nf-core.yml
@@ -1,7 +1,6 @@
 bump_version: null
 lint:
   files_unchanged:
-    - .github/workflows/linting.yml
     - lib/NfcoreTemplate.groovy
     - docs/images/nf-core-scnanoseq_logo_dark.png
     - docs/images/nf-core-scnanoseq_logo_light.png

--- a/.nf-core.yml
+++ b/.nf-core.yml
@@ -6,9 +6,6 @@ lint:
     - docs/images/nf-core-scnanoseq_logo_dark.png
     - docs/images/nf-core-scnanoseq_logo_light.png
     - .gitignore
-  pipeline_todos:
-    - README.md
-    - main.nf
   template_strings: false
 nf_core_version: 3.0.2
 org_path: null

